waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMappingImpl.java

```diff
@@ -525,6 +631,170 @@ public List<SchemaVersion> transformOutboundSchemaVersions(List<SchemaVersion> s
     return schemaVersions;
   }
 
+  @Override
+  public ISchemaName transformInboundISchemaName(ISchemaName iSchemaName) {
+    iSchemaName.setDbName(transformInboundDatabaseName(iSchemaName.getDbName()));
+    return iSchemaName;
+  }
+
+  @Override
+  public ISchemaName transformOutboundISchemaName(ISchemaName iSchemaName) {
```

Comment:
inline this method if only used in one place

Reply:
can we keep it in separate method so tht it will be clean?